tests/test_transform.py

```diff
@@ -131,5 +132,46 @@ def testCreatingQuantumGraphMixed(self):
         assert final.queue == "special_final_queue"
 
 
+class TestGetJobValues(unittest.TestCase):
+    """Tests of _get_job_values."""
+
+    def setUp(self):
+        self.gwjob = GenericWorkflowJob("default_job")
+
+    def testGettingDefaults(self):
+        """Test retrieving default values."""
+        config = BpsConfig({})
+        job_values = _get_job_values(config, {}, None)
+        assert all(
```

Comment:
The examples in the LSST developer guide are all _self.assert*_ functions.  Because using unittest, should they all be unittest's self.assert*?

Reply:
I'll modify tests to use methods provided by `unittest` package.